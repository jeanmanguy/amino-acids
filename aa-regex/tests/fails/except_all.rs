use aa_regex::*;

fn main() {
    let wrong = except!(A, R, N, D, C, E, Q, G, H, I, L, K, M, F, P, S, T, W, Y, V);
}
