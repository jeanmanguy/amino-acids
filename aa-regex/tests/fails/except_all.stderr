error: Cannot add more exceptions
 --> $DIR/except_all.rs:4:82
  |
4 |     let wrong = except!(A, R, N, D, C, E, Q, G, H, I, L, K, M, F, P, S, T, W, Y, V);
  |                                                                                  ^
